activerecord/lib/active_record/core.rb

```diff
@@ -98,6 +98,13 @@ def self.error_on_ignored_order_or_limit=(value)
         self.error_on_ignored_order = value
       end
 
+      # :singleton-method:
+      # Specify the behavior for unsafe raw query methods. A value of true will
```

Comment:
I went with `unsafe_raw` because 1) they are unsafe and 2) they allow raw SQL. I'd be happy to drop the `raw` bit if you'd like.